tasmota/language/pl-PL.h

```diff
@@ -247,46 +247,46 @@
 #define D_CONFIGURE_MQTT "Konfiguracja MQTT"
 #define D_CONFIGURE_DOMOTICZ "Konfiguracja Domoticza"
 #define D_CONFIGURE_LOGGING "Konfiguracja dziennika"
-#define D_CONFIGURE_OTHER "Konfiguracja innych"
+#define D_CONFIGURE_OTHER "Inne konfiguracje"
 #define D_CONFIRM_RESET_CONFIGURATION "Potwierdź reset ustawień"
 #define D_RESET_CONFIGURATION "Reset ustawień"
 #define D_BACKUP_CONFIGURATION "Kopia ustawień"
 #define D_RESTORE_CONFIGURATION "Przywracanie ustawień"
 #define D_MAIN_MENU "Menu główne"
 
 #define D_MODULE_PARAMETERS "Parametry modułu"
-#define D_MODULE_TYPE "Typ modułu"
+#define D_MODULE_TYPE "Typ"
 #define D_PULLUP_ENABLE "Przełącznik pull-up"
 #define D_ADC "ADC"
 #define D_GPIO "GPIO"
 #define D_SERIAL_IN "Wejście rs"
 #define D_SERIAL_OUT "Wyjście rs"
 
 #define D_WIFI_PARAMETERS "Parametry sieci WiFi"
-#define D_SCAN_FOR_WIFI_NETWORKS "Skanuj sieci WiFi"
+#define D_SCAN_FOR_WIFI_NETWORKS "Skanuj sieci"
 #define D_SCAN_DONE "Skanowanie wykonane"
 #define D_NO_NETWORKS_FOUND "Nie znaleziono sieci"
-#define D_REFRESH_TO_SCAN_AGAIN "Odswież aby ponownie wyszukać sieci WiFi"
-#define D_DUPLICATE_ACCESSPOINT "Powiel AP"
+#define D_REFRESH_TO_SCAN_AGAIN "Odśwież aby ponownie wyszukać sieci"
+#define D_DUPLICATE_ACCESSPOINT "Duplikuj"
 #define D_SKIPPING_LOW_QUALITY "Pominięto z powodu niskiej jakości sygnału"
 #define D_RSSI "RSSI"
 #define D_WEP "WEP"
 #define D_WPA_PSK "WPA PSK"
 #define D_WPA2_PSK "WPA2 PSK"
-#define D_AP1_SSID "AP1 SSID"
-#define D_AP1_PASSWORD "Hasło AP1"
-#define D_AP2_SSID "AP2 SSID"
-#define D_AP2_PASSWORD "Hasło AP2"
+#define D_AP1_SSID "Nazwa"
```

Comment:
Seems that D_AP1_SSID should be **"Nazwa 1"** and D_AP2_SSID "Nazwa 2"
The same goes to the SSID passwords